crates/astria-conductor/local.env.example

```diff
@@ -22,3 +22,6 @@ ASTRIA_CONDUCTOR_DISABLE_FINALIZATION=false
 
 # libp2p Port
 ASTRIA_CONDUCTOR_LIBP2P_PORT=2451
+
+# choose whether or not to execute empty blocks
+ASTRIA_CONDUCTOR_EXECUTE_EMPTY_BLOCKS=true
```

Comment:
We have `ASTRIA_CONDUCTOR_DISABLE_FINALIZATION` above.
Should this be something like `ASTRIA_CONDUCTOR_ENABLE_EMPTY_BLOCK_EXECUTION` to keep them more consistent?

I think it would be most ideal to have `ENABLE_FINALIZATION` and `ENABLE_EMPTY_BLOCK_EXECUTION`, but I don't think it's a blocker or anything.

Reply:
I think that boolean values should normally be false. If at some point in the future we walk back getting rid of clis, and add defaults back in this makes the most sense, setting the flag would be a deviation from normal. `true` values are essentially not the default (this also mimics boolean behaivior in many languages, uninitialized booleans are false).

In this instance I think that means we should make this a `disable`, I like `ASTRIA_CONDUCTOR_DISABLE_EMPTY_BLOCK_EXECUTION`.

Reply:
agreed w `ASTRIA_CONDUCTOR_DISABLE_EMPTY_BLOCK_EXECUTION` if we want to execute empty blocks by default

Reply:
updated to `ASTRIA_CONDUCTOR_DISABLE_EMPTY_BLOCK_EXECUTION`